### PR TITLE
chore: shaclc scope document

### PR DESCRIPTION
**Pull request by jeswr**
Per #36 

cc @VladimirAlexiev @HolgerKnublauch

### CODE REVIEW

**Review by HolgerKnublauch — APPROVED** [review]

File: shacl-compact-syntax/scope.md
Comment (TallTed) [reply shacl-compact-syntax/scope.md:5]: or...
```suggestion
 - Adding support for additional SHACLC features
```

Comment (TallTed) [reply shacl-compact-syntax/scope.md:5]: And then make all the other `SHACLC` and `SHACL-C` match.

I think `SHACL-C` is better.

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:35]:
```suggestion
There are attempts to implement lossless SHACL-C by allowing the use of a more Turtle-like syntax in places. We should align on a way of doing this and write it up in spec form. Existing attempts include:
 - [Extended Compact Syntax in `shaclcjs`](https://github.com/jeswr/shaclcjs?tab=readme-ov-file#extended-shacl-compact-syntax)
 - [make SHACL-C→SHACL nearly lossless TopQuadrant/shacl#98](https://github.com/TopQuadrant/shacl/issues/98) (escape into Turtle, and some extra links and ideas)
```

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:42]:
```suggestion
 - [SHACL-C: missing property shape TopQuadrant/shacl#142](https://github.com/TopQuadrant/shacl/issues/142) (test case)
 - [SHACL-C→SHACL outputs `;` instead of `.` TopQuadrant/shacl#91](https://github.com/TopQuadrant/shacl/issues/91) (test case)
 - [SHACL conversion: malformed lists TopQuadrant/shacl#92](https://github.com/TopQuadrant/shacl/issues/92) (test case)
```

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:48]: We cannot know that there are only these implementations; therefore `include` rather than `are`.
```suggestion
The current SHACL-C implementations include
 - https://jena.apache.org/documentation/shacl/#shacl-compact-syntax
 - https://github.com/jeswr/shaclcjs and https://github.com/jeswr/shaclc-writer
 - https://gitlab.com/allotrope-open-source/shape-editor
```

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:53]:
```suggestion
 - [SHACL-C grammar: nodeOr vs propertyOr #12](https://github.com/w3c/shacl/issues/12)
 - [Extending SHACL-C to enable the expression of any RDF statements #36](https://github.com/w3c/shacl/issues/36)
```

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:21]:
```suggestion
### Support for additional SHACL-C shorthands
```

File: shacl-compact-syntax/scope.md
Comment (TallTed) [inline shacl-compact-syntax/scope.md:31]:
```suggestion
### [Making SHACL-C Lossless](https://github.com/w3c/shacl/issues/36)
```

**Review by TallTed — COMMENTED** [review]
This is just one reason why it makes sense to leave PRs open for at least a day before merging... Extra eyes notice things.

File: shacl-compact-syntax/scope.md
Comment (jeswr) [reply shacl-compact-syntax/scope.md:5]: I for one do not have write access.

@TallTed on this occasion can I suggest you open a separate PR with your suggested changes?

**Comment by TallTed** [comment]
I will make a new PR.

I am also suggesting that going forward, PRs generally live for at least one business day (better two to three business days) before merge, because making change suggestions on a PR is far faster and easier than making a full PR.

**Pull request merged**
